src/qt/transactiontablemodel.cpp

```diff
@@ -30,6 +30,12 @@
 
 #define SINGLE_THREAD_MAX_TXES_SIZE 4000
 
+// Maximum amount of loaded records in ram in the first load.
+// If the user has more and want to load them, could add a load on demand in pages.
+// but.. to be realistic, it's not useful anyway, why someone would ever want to have more than 20,000 txs loaded in the UI??
+// The person would be essentially spamming himself..
```

Comment:
We definitely need to implement "load on demand" in the future; can think of several use cases requiring the whole txes loaded. That's why I would replace this monologue with a simple `TODO`.

Reply:
done